src/EnrollWebauthnController.js

```diff
@@ -79,7 +79,12 @@ function (Okta, Errors, FormType, FormController, CryptoUtil, webauthn, Footer,
               },
               excludeCredentials: getExcludeCredentials(activation.excludeCredentials)
             });
-            return new Q(navigator.credentials.create({publicKey: options}))
+            var webauthnAbortController = new AbortController();
+            this.appState.set('webauthnAbortController', webauthnAbortController);
```

Comment:
do this really belong to the `appState`? Can we instead abort in the Footer when the user clicks "Back to Factors"? It seems to me this is very specific to this factor